msal/changelog.txt

```diff
@@ -1,3 +1,10 @@
+2.3.0-preview
```

Comment:
>2.3.0 [](start = 0, length = 5)

2.3.0 for ADAL consistency and because the API was extended?

Reply:
Error codes are an addition to the public API. The semantic versioning is a bit unclear on when to upgrade the minor version, so I suppose it could go either way. 

Minor version Y (x.Y.z | x > 0) MUST be incremented if new, backwards compatible functionality is introduced to the public API. It MUST be incremented if any public API functionality is marked as deprecated. It MAY be incremented if substantial new functionality or improvements are introduced within the private code. It MAY include patch level changes. Patch version MUST be reset to 0 when minor version is incremented.